src/main/scala/uk/gov/nationalarchives/tdr/api/service/FileMetadataService.scala

```diff
@@ -24,6 +24,10 @@ class FileMetadataService(fileMetadataRepository: FileMetadataRepository,
 
   val loggingUtils: LoggingUtils = LoggingUtils(Logger("FileMetadataService"))
 
+  def getSumOfFileSizes(consignmentId: UUID): Future[Int] = {
+    fileMetadataRepository.getSumOfFileSizes(consignmentId)
```

Comment:
I didn't originally do one because all it does is call the repository method and that's got quite a few tests for it but I've added one in for coverage.